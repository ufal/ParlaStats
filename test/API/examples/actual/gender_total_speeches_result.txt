[{'person.sex': '', 'total_speeches': 3}, {'person.sex': 'M', 'total_speeches': 4405}, {'person.sex': 'F', 'total_speeches': 1283}]

Running time 5.25048828125 ms

+------------+----------------+
| person.sex | total_speeches |
+------------+----------------+
|            |       3        |
|     M      |      4405      |
|     F      |      1283      |
+------------+----------------+
