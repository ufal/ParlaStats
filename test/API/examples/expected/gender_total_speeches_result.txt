[{'person.sex': '', 'total_speeches': 3}, {'person.sex': 'M', 'total_speeches': 4405}, {'person.sex': 'F', 'total_speeches': 1283}]

Running time 12.240478515625 ms

+------------+----------------+
| person.sex | total_speeches |
+------------+----------------+
|            |       3        |
|     M      |      4405      |
|     F      |      1283      |
+------------+----------------+
